client/lib/abtest/active-tests.js

```diff
@@ -3,10 +3,10 @@ export default {
 	improvedOnboarding: {
 		datestamp: '20190214',
```

Comment:
Can we go ahead and bump the date with this test so that we can have clear metrics around users in the test pool? This will require that the test slug in the e2e's gets updated too.

Reply:
Already done!